camaleon_cms.gemspec

```diff
@@ -32,7 +32,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'will_paginate-bootstrap'
   s.add_dependency 'breadcrumbs_on_rails'
   s.add_dependency 'font-awesome-rails'
-  s.add_dependency 'tinymce-rails', '~> 4.3'
+  s.add_dependency 'tinymce-rails', '~> 4.2.5'
```

Comment:
@superchell why this change?

Reply:
@owen2345 sorry, this commit shouldn't have gotten into this PR

Reply:
@superchell 
I understand, please update the pull request to approve it.

Reply:
@owen2345 done!